canonical-websites/www.ubuntu.com
# Correct the USN redirect

USN will be made available at https://usn.ubuntu.com/usn/, rather than https://usn.ubuntu.com/ as we previously thought. This means this redirect needs to be updated, ready for when we turn off the existing routing behind https://www.ubuntu.com/usn/

## QA

``` bash
./run
```

Browse to `http://127.0.0.1:8001/usn/something` and check it redirects you to `https://usn.ubuntu.com/usn/something` (even though it will 404).
